test/test_with_voyager.py

```diff
@@ -21,13 +21,13 @@
 from hyperseti import find_et
 from hyperseti.log import update_levels
 
-from file_defs import synthetic_fil, test_fig_dir, voyager_h5
+from .file_defs import synthetic_fil, test_fig_dir, voyager_h5
```

Comment:
This relative import error is quite frustrating when testing (at least with my approach of running from the test directory on occasion).

Wondering whether test data should be moved elsewhere? In PyGDSM I opted to use astropy's download utility, which downloads to `~/.astropy/cache` by default:
https://github.com/telegraphic/pygdsm/blob/52eff5cd875d0938211201a8b3625b5b3836ec5d/pygdsm/component_data.py#L1

We could change download path to `~/.hyperseti`, or use a relative path to test directory. 

The current location of `/test` is the most logical for command-line development, but I think `pip install` may actually treat 'test' as a package and install it into `site-packages`. So could think about moving too

Reply:
I have always detested the inconsistency of "from .FILE" syntax in Python.  It works for pytest but it fails if you execute the file directly with python3.  In turbo_seti newer tests, I put them under the termporary directory, like this:
```
from tempfile import gettempdir
TESTDATA = gettempdir() + "/hyperseti_test_data/"
```
We could do that with test_data and test_figs in general.